crates/consensus/src/signed.rs

```diff
@@ -32,6 +32,11 @@ impl<T, Sig> Signed<T, Sig> {
     pub fn into_parts(self) -> (T, Sig, B256) {
         (self.tx, self.signature, self.hash)
     }
+
+    /// Returns the transaction.
+    pub fn owned_tx(self) -> T {
```

Comment:
i would prefer this to be something like `strip_signature` to make it clear that we're dropping the sig